integration-tests/downscope/downscope_test.go

```diff
@@ -141,14 +141,15 @@ func helper(t *testing.T, tt downscopeTest) error {
 	if err != nil {
 		return fmt.Errorf("failed to create the storage service: %v", err)
 	}
-	obj, err := storageService.Objects.Get(bucket, tt.objectName).Download()
+	resp, err := storageService.Objects.Get(bucket, tt.objectName).Download()
 	if err != nil {
 		return fmt.Errorf("failed to retrieve object from GCP project with error: %v", err)
 	}
-	_, err = ioutil.ReadAll(obj.Body)
+	_, err = ioutil.ReadAll(resp.Body)
 	if err != nil {
+		resp.Body.Close()
```

Comment:
Sorry if I was unclear, I meant to say this should be defered after line 147, before it has been read. That will cover all cases.